data/json/mapgen_palettes/oil_platform_palette.json

```diff
@@ -0,0 +1,307 @@
+        { "item": "toasterpastryfrozen_box_snack_2", "chance": 80, "repeat": [ 1, 3 ] }
+      ],
+      "ç": [
+        { "item": "kitchen_nonfood", "chance": 80, "repeat": [ 0, 5 ] },
+        { "item": "kitchen_appliances", "chance": 80, "repeat": [ 0, 5 ] },
+        { "item": "crate_kitchen", "chance": 80, "repeat": [ 0, 5 ] },
+        { "item": "kitchen_counters", "chance": 80, "repeat": [ 0, 5 ] },
+        { "item": "groce_condiment", "chance": 80, "repeat": [ 2, 5 ] }
+      ],
+      "4": [ { "item": "SUS_oven", "chance": 85, "repeat": [ 0, 4 ] } ],
+      "≡": [ { "item": "SUS_dishwasher", "chance": 85, "repeat": [ 0, 4 ] } ],
+      "♫": { "item": "fuel_drums", "chance": 30 },
+      "`": { "item": "oil_drums", "chance": 40 },
+      "▀": { "item": "pool_table", "chance": 60 }
+    },
+    "vehicles": {
+      "Ƨ": { "vehicle": "swivel_chair", "chance": 100, "rotation": [ 0, 90, 180, 270 ] },
+      "²": { "vehicle": "forklift", "chance": 100, "rotation": [ 0, 90, 180, 270 ] },
+      "→": { "vehicle": "helicopters", "chance": 20, "rotation": [ 0, 90, 180, 270 ] },
+      "n": { "vehicle": "inflatable_boat", "chance": 80 }
+    },
+    "vendingmachines": { "D": { "item_group": "vending_drink_items" }, "F": { "item_group": "vending_food_items" } },
+    "monster": {
+      "i": { "monster": "mon_zombie_swimmer_base", "chance": 30 },
+      "↨": { "monster": "mon_gas_zombie", "chance": 5 },
+      "←": { "monster": "mon_zombie_scientist", "chance": 70 }
+    }
+  },
+  {
+    "id": "fuel_drums",
+    "type": "item_group",
+    "items": [
+      { "item": "crude_oil", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 40 },
+      { "item": "motor_oil", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 40 },
+      { "item": "lamp_oil", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 40 },
+      { "item": "gasoline", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 50 },
+      { "item": "diesel", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 40 },
+      { "item": "chem_benzene", "container-item": "55gal_drum", "charges-min": 26, "charges-max": 126, "prob": 40 },
+      { "item": "crude_oil", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 40 },
+      { "item": "motor_oil", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 40 },
+      { "item": "lamp_oil", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 40 },
+      { "item": "gasoline", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 50 },
+      { "item": "diesel", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 40 },
+      { "item": "chem_benzene", "container-item": "metal_tank", "charges-min": 26, "charges-max": 55, "prob": 40 },
```

Comment:
It doesn't really make a lot of sense for refined products to be here, let alone in half-filled containers. Besides the scarcity of east coast oil platforms, all the refining capacity is in coastal plants. The platform would either be pumping the gas directly to shore or offloading onto cargo ships for a short jaunt back to the existing refineries.

I recommend keeping the empty drum chance, keep the crude and making sure the crude is always full, then removing all the distillates.

Reply:
You're totally right, I had a vague idea about fuel for the forklift and all, but that shouldn't be here either. Neither made sense, they're gone now. I kept some motor oil around as lubricant is incredibly important with such machinery, and didn't put too many crude oil drums around since the system I'm trying to represent would be one where it's directly pipe-lined away, so those drums would mostly be for sampling. Thank you again !